depends/packages/native_cxxbridge.mk

```diff
@@ -20,8 +20,8 @@ $(package)_patches=Cargo.lock
 $(package)_extra_sources=$(package)-$($(package)_version)-vendored.tar.gz
 
 define $(package)_fetch_cmds
-$(call fetch_file,$(package),$($(package)_download_path),$($(package)_download_file),$($(package)_file_name),$($(package)_sha256_hash)) && \
-$(call vendor_crate_deps,$(package),$($(package)_file_name),$(PATCHES_PATH)/$(package)/Cargo.lock,Cargo.toml,$(package)-$($(package)_version)-vendored.tar.gz)
+$(call fetch_file,$(1),$($(1)_download_path),$($(1)_download_file),$($(1)_file_name),$($(1)_sha256_hash)) && \
+$(call vendor_crate_deps,$(1),$($(1)_file_name),$(PATCHES_PATH)/$(1)/Cargo.lock,Cargo.toml,$(1)-$($(1)_version)-vendored.tar.gz)
```

Comment:
When we call `native_cxxbridge_fetch_cmds` from `rustcxx.mk`, what will `$(package)` be set to? If it is dynamically scoped then I understand, but yuck.

Reply:
`$(package)` is set to `rustcxx` in that context. `$(call)` only substitutes numberic substitutions with the given parameters, hence making this change.